packages/eslint-config-kodefox/index.js

```diff
@@ -54,7 +54,18 @@ module.exports = {
         noUselessIndex: true,
       },
     ],
-    'import/order': ['warn', { 'newlines-between': 'always' }],
+    'import-helpers/order-imports': [
+      'warn',
+      {
+        newlinesBetween: 'always',
+        groups: [
+          'module',
+          ['parent', 'sibling', 'index'],
+          ['//use[A-Z].*/', '//helper(s?)/'],
+          ['//constant(s?)/', '/type(s?)/'],
```

Comment:
Yes, because sometimes we didn't put `types.ts` inside `types` directory.

Reply:
And also could be there is a chance that someone using  `something.types.ts` or `something.type.ts`.

Reply:
Usually when a regex is in string form it doesn't have the `/` on each side (that is a syntax for the regex literal) and thus doesn't need those characters to be escaped since they don't have special meaning. So in this case are the `/` intended to match actual slashes? In that case why are there double at the beginning? (Maybe this plugin uses a weird form of regex string?)

Reply:
From the plugin example, it uses `/` on each side of the regex string so I follow the pattern.

The reason why there is double `/` at the beginning is that the second one is intended to match the `/` from directory path.